projects/cds/src/merchandising/adapters/strategy/cds-merchandising-strategy.adapter.spec.ts

```diff
@@ -32,9 +33,24 @@ const STRATEGY_PRODUCTS: StrategyProducts = {
   },
 };
 const STRATEGY_REQUEST = {
```

Comment:
you can keep the complex object lower case (same case to all that has uppercase)

Reply:
Done